sentry_sdk/integrations/wsgi.py

```diff
@@ -78,17 +78,22 @@ def __call__(self, environ, start_response):
         hub = Hub(Hub.current)
 
         with hub:
+            span = None
             with capture_internal_exceptions():
                 with hub.configure_scope() as scope:
                     scope.clear_breadcrumbs()
                     scope._name = "wsgi"
-                    scope.set_span_context(SpanContext.continue_from_environ(environ))
                     scope.add_event_processor(_make_wsgi_event_processor(environ))
+                    scope.span = span = Span.continue_from_environ(environ)
```

Comment:
`continue_from_environ` could live in this integration instead as a private function?

Reply:
I decided against this, these are common helper methods that construct spans. It's fine to have them as equal to other constructors.